presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -2601,6 +2602,19 @@ public boolean isNativeExecutionEnabled()
         return this.nativeExecutionEnabled;
     }
 
+    @Config("disable-timestamp-with-timezone-for-native-execution")
```

Comment:
@gggrace14 @kgpai @kagamiori For testing the timestamp comparison features, we will need to turn this on in deployment configs.